lib/Phpfastcache/Drivers/Mongodb/Driver.php

```diff
@@ -224,19 +224,31 @@ protected function collectionExists($collectionName): bool
      */
     protected function buildConnectionURI($databaseName = ''): string
     {
+        $servers = $this->getConfig()->getServers();
+        $options = $this->getConfig()->getOptions();
+
         $host = $this->getConfig()->getHost();
         $port = $this->getConfig()->getPort();
         $username = $this->getConfig()->getUsername();
         $password = $this->getConfig()->getPassword();
 
+        if( \count($servers) > 0 ){
+            $host = array_reduce($servers, function($carry, $data){
```

Comment:
Please force the force the namespace using "\array_reduce" to improve opcache efficiency :)

Reply:
It's a mistake! I edited and pushed! :)